### DISCUSSION TITLE
Group.id is not used

### DISCUSSION THREAD

**Discussion by verbotenj** [strimzi/strimzi-kafka-operator#10476] [General]
I have a default setup for KafkaConnect with `group.id: connect-cluster`
config:
   group.id: connect-cluster
   
When I try to explore consumer groups, I get something similar to the default name cluster name.
./kafka-consumer-groups.sh --list --bootstrap-server $BROKER
connect-kafka-kafka-cluster...

Why my `group.id` is not used?

**Comment by scholzj** [comment]
It is used internally by Connect: https://kafka.apache.org/documentation/#connectconfigs_group.id ... I'm not entirely sure if you are expected to see any committed offsets for it. It might be used for coordination purposes for example.

**Reply by verbotenj** [reply]
I was expecting to describe my group, but I got 

Consumer group 'Group name' has no active members.

When I removed `group.id`. 
I was able to describe the group and got the table of topics.

```
./kafka-consumer-groups.sh --bootstrap-server $BROKER --group $GROUP --describe

GROUP                                                TOPIC                                                                  PARTITION  CURRENT-OFFSET  LOG-END-OFFSET  LAG             CONSUMER-ID                                                                                            HOST            CLIENT-ID
 
 ```
 
 I am unsure if `group.id` = `consumer group` because I didn't get `connect-cluster` when I ran—-list.
connect-cluster != connect-kafka-kafka-cluster..
 
 https://github.com/strimzi/strimzi-kafka-operator/blob/e9ea2380716020ec0baa12d6b314966feb0da06f/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectConfiguration.java#L28

**Reply by scholzj** [reply]
I'm really sorry, but I don't follow what you mean. As I said, the `group.id` is used by Connect and it is up to Connect how does it use it.

**Reply by scholzj** [reply]
BTW: In my cluster, I also do not see any commits done by the Connect group ID. But as I said, I do not think it indicates any issues, it is just that the group is used for other purposes by Connect.

**Reply by verbotenj** [reply]
Thank you
